src/Features/Core/Portable/Common/GlyphTags.cs

```diff
@@ -3,7 +3,7 @@
 using System.Collections.Immutable;
 using Microsoft.CodeAnalysis.Tags;
 
-namespace Microsoft.CodeAnalysis.Completion
+namespace Microsoft.CodeAnalysis
```

Comment:
Just checking: any chance this is used by TypeScript/F# and isn't something they could easily deal with?

(My presumption is that we'll move F#/TypeScript directly to editor APIs for quick info to prevent you from having breaking changes, but this might cause problems elsewhere.)

Reply:
Regardless, we update F#/TypeScript for breaking changes like this when we move them directly to the editor APIs, no? This is clearly in the wrong namespace.

Reply:
And to specifically answer your question: *Of course* there's a chance this is used by TypeScript/F#. They can be using *anything* that's internal.